site/content/contribute/team_contributions/_index.md

```diff
@@ -39,7 +39,9 @@ Translate Mattermost into a new language or help maintain an existing one by joi
 
 ## Documentation Contributors
 
-To be added.
+Documentation contributors by GitHub username and (first contribution date) are alphabetically listed below. The list is updated regularly (last updated June 28, 2020).
```

Comment:
```suggestion
Documentation contributors by GitHub username and (first contribution date) are alphabetically listed below. The list is updated regularly (last updated June 28, 2020), and includes contributions to [docs](https://github.com/mattermost/docs) repository.

Join others who have contributed documentation for Mattermost, and [get started today](https://github.com/mattermost/docs#contributing).
```

Reply:
^there may be a better "get started" guide you have available today. I took the link we have at https://mattermost.com/contribute/ for Documentation Contributors.

Reply:
Ah thanks for adding this! I'd meant to comment and mention I didn't know what else to add. That links works best - I'll update the style guide link but otherwise I'm happy for that to be used.